packages/image-block/src/ImageBlock.spec.ct.tsx

```diff
@@ -50,8 +51,12 @@ describe('Image Block', () => {
         cy.get(ImageBlockImageSelector).should('exist');
     });
 
-    it('should render the download button if the image is uploaded', () => {
+    it('should render the download button if the image is uploaded and security settings allows it', () => {
```

Comment:
Can you add an extra test case checking if the security does not allow it?

Reply:
also added one more test here